packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -146,8 +146,12 @@ function ViewList( {
 } ) {
 	const columns = useMemo( () => {
 		const _columns = fields.map( ( field ) => {
-			const { render, ...column } = field;
-			column.cell = ( props ) => render( props.row.original, view );
+			const { render, getValue, ...column } = field;
+			column.cell = ( props ) =>
```

Comment:
We should also cover against `field.render` not existing, e.g.: [status](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/page-pages/index.js#L167).

Reply:
This happens in the parent component: https://github.com/WordPress/gutenberg/pull/55434/files#diff-a72d10c309e831c5e03caff0c1b9cfb82885abb105377a196cca642ea4d711adR32

Reply:
ok, that's confusing 😅 Can we normalize both things in the same place?

Reply:
>A follow-up we may need to do is the following: the tanstack API may use the presence of accessorFn and cell to determine whether a feature is enabled or not

Exactly that. Let's do that in a separate PR and see if we can remove the `getValue` completely.